packages/atomic/cypress/integration/facets/facet-common-actions.ts

```diff
@@ -11,31 +11,103 @@ export function selectIdleCheckboxValueAt(
   FacetWithCheckboxSelector: FacetWithCheckboxSelector,
   index: number
 ) {
-  FacetWithCheckboxSelector.idleCheckboxValue().eq(index).click();
+  FacetWithCheckboxSelector.idleCheckboxValueLabel()
+    .eq(index)
+    .then((idleCheckboxValueLabel) => {
+      const hasAriaLabel = !!idleCheckboxValueLabel.attr('aria-label');
+      const text = hasAriaLabel
+        ? idleCheckboxValueLabel.attr('aria-label')!
+        : idleCheckboxValueLabel.text();
+      FacetWithCheckboxSelector.idleCheckboxValueLabel()
+        .filter(
+          hasAriaLabel ? `[aria-label="${text}"]` : `:contains("${text}")`
+        )
+        .its('length')
+        .should(
+          'eq',
+          1,
+          'There should not be any other value similar to this one.'
+        );
```

Comment:
This assertion is not for the purpose of waiting, I added it because I expect we could get some flakiness with `:contains(TEXT)` if we select multiple values that are similar.

E.g.,: if we selected "sewing table" then selected "table", then I think the last assertion would pass immediately without waiting for re-render.

I prefer having this assertion to prevent even the possibility of introducing a test case that is flaky like that.